### ISSUE TITLE
PHEDEX::Core::DB &amp;dbprep should optionally prefix schema objects

### ISSUE THREAD

**Issue by ericvaandering** [dmwm/PHEDEX#202]
Original Savannah ticket 22467 reported by None on Sat Dec 16 09:39:16 2006.

To avoid the problem experienced in summer with which schema a cursor was attached to, &amp;dbprep() in UtilsDB should optionally rewrite the SQL to automatically add a schema prefix.  Something like 'SchemaPrix cms_transfermgmt.' in the DBParam file, and then replace anything with a space followed by 't', 'seq', 'fk', 'pk', 'uq', 'ck', 'ix', 'tr', and a underscore, by injecting the prefix at that point.

In other words it should automatically convert &quot;insert into t_foo&quot; into &quot;insert into cms_transfermgmt.t_foo&quot; if the DBParam says so.
